src/app/shared/filter-bar/filter-bar.component.html

```diff
@@ -39,7 +39,16 @@
       </mat-form-field>
       <mat-form-field appearance="standard">
         <mat-label>Milestone</mat-label>
-        <mat-select #milestoneSelectorRef [(value)]="this.dropdownFilter.milestones" (selectionChange)="applyDropdownFilter()" multiple>
+        <mat-select
+          #milestoneSelectorRef
+          [(value)]="this.dropdownFilter.milestones"
+          (selectionChange)="applyDropdownFilter()"
+          [disabled]="this.milestoneService.hasNoMilestones"
+          multiple
+        >
+          <mat-select-trigger *ngIf="this.milestoneService.hasNoMilestones">
+            <span>No Milestones</span>
+          </mat-select-trigger>
```

Comment:
Hi! I think it might be a bit odd to see `Without a milestone` while the select is disabled, which was why I changed it to display `No Milestones` instead which has a clearer message.

But, I'm definitely open to reverting this.

Reply:
Hmm, that seems fine for now!